CHANGELOG.md

```diff
@@ -0,0 +1,16 @@
+# Changelog
+All notable changes to this project will be documented in this file.
+
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
+and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
+
+## [Unreleased]
+
+## [0.0.1] - 2022-10-21
+### Added
+- This CHANGELOG file
+
+### Changed
+- Moved from https://github.com/anexia-it/django-aws-iot to https://github.com/anexia/django-aws-iot
```

Comment:
We do not mention the move between organisations if not needed. However, we may want to add "Initial release" under "Added".